app/score_calculator/yaku_check/block_yaku/mixed_double_chow.py

```diff
@@ -0,0 +1,22 @@
+from typing import Final
+
+from app.score_calculator.block.block import Block
+from app.score_calculator.enums.enums import BlockType
+
+SIZE: Final[int] = 2
+TYPE_COUNT: Final[int] = 2
+
+
+def valid_check_mixed_double_chow(blocks: list[Block]) -> bool:
+    if len(blocks) != SIZE:
+        return False
+    return all(block.type == BlockType.SEQUENCE for block in blocks)

```

Comment:
len(blocks)!=Size 일때 오류를 발생시키던가
단순하게 구현합시다.
```suggestion
    return len(blocks) == SIZE and all(block.type == BlockType.SEQUENCE for block in blocks)

```